packages/fluentui/react-northstar/src/components/Pill/Pills.tsx

```diff
@@ -53,12 +53,15 @@ export const Pills: ComponentWithAs<'div', PillsProps> & FluentComponentStaticPr
 
   const ElementType = getElementType(props);
   const unhandledProps = useUnhandledProps(Pills.handledProps, props);
+  const pillBehavior = React.useMemo(() => getA11Props.unstable_behaviorDefinition().childBehaviors.pill, [
+    getA11Props,
+  ]);
```

Comment:
```suggestion
  const pillBehavior = getA11Props.unstable_behaviorDefinition().childBehaviors.pill;
```

- there is no need to memo it a we have `childProps` below that are stable
- `getA11Props()` in `useAccessibility` is not referentially stable